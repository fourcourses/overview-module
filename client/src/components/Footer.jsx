import React from 'react';

const Footer = () => {
  return (
    <div className="footer">
    Support
    </div>
  )
}

export default Footer;