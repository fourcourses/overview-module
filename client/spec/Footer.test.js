import React from 'react';
import { shallow, mount } from 'enzyme';

import Footer from '../src/components/Footer.jsx';


describe('Header', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Footer debug/>);

    expect(component).toMatchSnapshot();
  })

  it('should render correctly with no props', () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  })
});
