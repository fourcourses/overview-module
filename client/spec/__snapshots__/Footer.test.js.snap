// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`Header should render correctly with no props 1`] = `ShallowWrapper {}`;
